### ISSUE TITLE
Array to string conversion when using story link types

### ISSUE THREAD

**Issue by Axel29** [storyblok/storyblok-php-richtext-renderer#10]
### Steps to reproduce (*)

1. Create a story
2. Add a Richtext content
3. Add an internal link (pointing to a story) inside the Richtext content

### Expected result (*)

1. The link is interpreted when using the renderer

### Actual result (*)

1. A `Twig\Error\RuntimeError` Exception is thrown wit the following message: `An exception has been thrown during the rendering of a template ("Notice: Array to string conversion in /app/vendor/storyblok/richtext-resolver/src/Utils/Render.php on line 51"). in /app/storyblok/components/seo.twig:6`

You can find the stack trace bellow:

```
Next Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("Notice: Array to string conversion in /app/vendor/storyblok/richtext-resolver/src/Utils/Render.php on line 51"). in /app/storyblok/components/seo.twig:6
Stack trace:
#0 /app/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling(Array, Array)
#1 /app/vendor/twig/twig/src/Environment.php(418) : eval()'d code(55): Twig\Template->display(Array)
#2 /app/vendor/twig/twig/src/Template.php(405): __TwigTemplate_493c729aecad815840c30a83c1fadb863275f1ecd68e9e25c50b8a8990f2e6e1->doDisplay(Array, Array)
#3 /app/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling(Array, Array)
#4 /app/vendor/twig/twig/src/Environment.php(418) : eval()'d code(37): Twig\Template->display(Array)
#5 /app/vendor/twig/twig/src/Template.php(405): __TwigTemplate_7a73c96e6d311c5d8ccb013e0fcc67c3713f103f067b6a080ba8cfd044cdb97b->doDisplay(Array, Array)
#6 /app/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling(Array, Array)
#7 /app/vendor/twig/twig/src/Template.php(390): Twig\Template->display(Array)
#8 /app/vendor/twig/twig/src/TemplateWrapper.php(45): Twig\Template->render(Array, Array)
```

**Comment by LSERRE** [comment]
Having the same issue with a link in a richtext content

**Comment by pscheit** [comment]
this should fix it #11 

buuuut, it seems to only occur, if you have "resolve_relations" set to resolve stories

**Comment by pscheit** [comment]
its fixed, release is 2.0.0

**Comment by roberto-butti** [comment]
@ademarCardoso, @joaokamun  I think we can close this issue
